src/Filesystem.php

```diff
@@ -67,4 +67,30 @@ public static function findFilesRecursively(string $path, string $pattern): arra
 
         return $found;
     }
+
+    public static function getLastModified(string $path): int
```

Comment:
I'll move these two functions and the related changes into a separate PR - that should make things cleaner.